lib/cc/engine/analyzers/analyzer_base.rb

```diff
@@ -13,7 +13,9 @@ class Base
           ::RuntimeError,
         ].freeze
 
-        BASE_POINTS = 1_500_000
+        POINTS_PER_MINUTE = 10_000 # Points represent engineering time to remediate
+        HOUR_POINTS = 60 * POINTS_PER_MINUTE
+        BASE_POINTS = HOUR_POINTS / 2
```

Comment:
FWIW, this is a bit too fancy for my tastes. `30 * POINTS_PER_MINUTE` is perfectly clear IMO. Also `HOUR_POINTS` is grammatically odd and inconsistent (s/b `POINTS_PER_HOUR`).